Packs/SentinelOne/ModelingRules/SentinelOneModelingRules/SentinelOneModelingRules.xif

```diff
@@ -87,5 +87,11 @@ filter eventType = "Alert"
     xdm.target.file_before.path = json_extract_scalar(targetProcessInfo, "$.tgtFileOldPath"),
     xdm.target.process.command_line = json_extract_scalar(targetProcessInfo, "$.tgtProcCmdLine"),
     xdm.target.process.executable.filename = json_extract_scalar(targetProcessInfo, "$.tgtProcName"),
-    xdm.target.process.integrity_level = to_integer(json_extract_scalar(targetProcessInfo, "$.tgtProcIntegrityLevel")),
+    xdm.target.process.integrity_level = if(json_extract_scalar(targetProcessInfo, "$.tgtProcIntegrityLevel") contains "protected", 20480, 
```

Comment:
create a new variable on line 56:
lower_tgtProcIntegrityLevel = lowercase(json_extract_scalar(targetProcessInfo, "$.tgtProcIntegrityLevel")

to be used in this line (90):
```suggestion
    xdm.target.process.integrity_level = if(lower_tgtProcIntegrityLevel contains "protected", 20480,
```